SBOMDocumentTypes.md

```diff
@@ -0,0 +1,30 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SBOMDocumentTypes
+
+## Summary
+
+Provides a set of values to be used to describe the common types of SBOMs that tools may create.
+
+## Description
+
+This field provides a list with agreed on definitions of SBOM types, as defined in [Types of Software Bill of Material (SBOM) Documents](https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf), published on April 21, 2023. 
+This field is a reasonable estimate of the most likely type of an SBOM from the producer and consumer perspective from which both parties can draw conclusions
+about the data inside an SBOM.  A single SBOM can have multiple SBOM document types associated with it.
+
+## Metadata
+
+- name: SBOMDocumentTypes
+
+## Entries
+
+- design: SBOM of intended, planned software project or product with included components (some of which may not yet exist) for a new software artifact.
+- source: SBOM created directly from the development environment, source files, and included dependencies used to build an product artifact. 
+- build: SBOM generated as part of the process of building the software to create a releasable artifact (e.g., executable or package) from data such as source files, dependencies, built
+components, build process ephemeral data, and other SBOMs.
```

Comment:
```suggestion
- build: SBOM generated as part of the process of building the software to create a releasable artifact (e.g., executable or package) from data such as source files, dependencies, build
components, build process ephemeral data, and other SBOMs.
```
This is more of a question if this "t" is intentional or should be a "d".

Reply:
In either case, `build` or `built` doesn't make much sense as a single word in this sentence... "From data such as build?" Seems like it's missing a word (build logs?)

Reply:
@rnjudge - it was a bad line split.    Should be "built components"    Want to retain it as is, as that's what in the publication.